### PR TITLE
Feat/add first&last name to employee

### PR DESCRIPTION
**Pull request by mdobosz3**
Existing employee can add their first and last name.
Each new employee must provide their first and last name - validation is imposed
The administrator can change all employee data.

PS while doing this task I also did an overdue task with the RUD implementation of employee data by the admin

### CODE REVIEW

**Comment by mdobosz3** [comment]
I changed the migration in the database - I deleted the default value and null: false.
Admin can edit employees who entered their full name. If the employee does not fill in the data, admin will not be able to add new kudos points. :)

File: app/controllers/employees_controller.rb [hunk 1]
```diff
@@ -25,7 +24,7 @@ def update
   end
 
   def check_employee
-    return if current_employee.first_name == 'default' && current_employee.last_name == 'default'
+    return if current_employee.first_name.nil? && current_employee.last_name.nil?
```
Comment (rniedziolka) [inline app/controllers/employees_controller.rb:27]: I think it could be moved to the policy, but this is my reflection ;)

Comment (oskarlakner) [reply app/controllers/employees_controller.rb:27]: good catch @rniedziolka ! <3

File: app/controllers/employees_controller.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  def show
+    render :show, locals: { employee: current_employee }
+  end
+
+  def edit
+    render :edit, locals: { employee: current_employee }
+  end
+
+  def update
+    params[:employee].compact_blank! if params[:employee][:email].blank? && params[:employee][:password].blank?
```
Comment (oskarlakner) [inline app/controllers/employees_controller.rb:16]: mutating params hash is a bad practice. I'd consider creating a method called `update_employee_params` and here perform these operations using maybe `.tap` method

**Review by oskarlakner — APPROVED** [review]
🍻

File: app/controllers/employees_controller.rb (hunk 1 above)
Comment (mdobosz3) [reply app/controllers/employees_controller.rb]: I took it to the policy :)

File: app/controllers/employees_controller.rb (hunk 2 above)
Comment (mdobosz3) [reply app/controllers/employees_controller.rb]: I also used policy to modify params

**Pull request merged**